### ISSUE TITLE
Typescript tries to load html-templates as modules

### ISSUE THREAD

**Issue by ijager** [bsliran/angular2-meteor-base#20]
When running this project freshly cloned in Meteor 1.4 I get these messages on the command line:

```
=> Started proxy.
=> Started MongoDB.
client/imports/demo/demo.component.ts (6, 22): Cannot find module './demo.component.html'.
client/app.component.ts (5, 22): Cannot find module './app.component.html'.
=> Started your app.

=> App running at: http://localhost:3000/
```

Also the typescript compiler of VSCode says `Cannot find module './demo.component.html'`

The project does work, but the compiler expects a module instead of a template?

**Comment by kamilkisiela** [comment]
It's a known issue:
- Urigo/meteor-static-templates#9
- Urigo/angular2-meteor#352

Since a component doesn't exist without its template, in our opinion it is the best approach to import the content.
This is why we created [`urigo:static-templates`](https://github.com/Urigo/meteor-static-templates).

`angular2-compilers` is based on the same compiler as the `urigo:static-templates`.

As I said in the first issue I linked above, it will be solved once we update `barbatus:typescript` package to TypeScript 2.0.

For now, you can update a TS plugin in your editor (if it's possible) to be using TypeScript 2.0 and then add code below to disable those warnings:

``` ts
declare module "*.html" {
    const template: string;
    export default template;
}
```

I don't know if it will help but you can try it :) This way your IDE won't be displaying any warnings.

**Comment by ijager** [comment]
Thanks for the IDE workaround. I'll wait for TS2.0 then :)

**Comment by kamilkisiela** [comment]
@ijager Should be fixed with #22 :+1:

**Comment by ijager** [comment]
It works now. Thanks
